docs/client.rst

```diff
@@ -72,18 +72,111 @@ WBEM Operation recording
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
 The WBEM client library API provides the possibility to record the WBEM
-operations that are executed on a connection. This is disabled by default
-and can be enabled by setting the
-:attr:`~pywbem.WBEMConnection.operation_recorder` instance variable of the
-:class:`~pywbem.WBEMConnection` object to an operation recorder object,
-i.e. to an object of a subclass of :class:`~pywbem.BaseOperationRecorder`.
-
-Typical usage scenarios for operation recorders are the tracing of WBEM
-operations, or the generation of test cases.
-
+operations that are executed on a connection.
 WBEM Operation recording uses the classes and subclasses defined in
 :ref:`WBEM Operation Recorder`.
 
+This is disabled by default and can be enabled by adding recorders to a
+WBEMCOnnection with the method
```

Comment:
Typo in spelling of `WBEMCOnnection` class name, plus link to the class with:
```
:class:`~pywbem.WBEMConnection` object
```

Reply:
Done